.github/workflows/ci.yml

```diff
@@ -285,8 +285,15 @@ jobs:
       - run: python -m pytest -n 2 modin/pandas/test/test_general.py --backend=${{ matrix.backend }}
       - run: |
           curl -o codecov https://codecov.io/bash
-          curl -o SHA512SUM https://raw.githubusercontent.com/codecov/codecov-bash/master/SHA512SUM
-          if sha512sum -c --ignore-missing --status SHA512SUM; then
+          VERIFIED_SCRIPT=true
+          VERSION=$(grep -o 'VERSION=\"[0-9\.]*\"' codecov | cut -d'"' -f2);
+          for i in 1 256 512
```

Comment:
what is the point of verifying all possible checksums? one should be enough...

Reply:
I was about to write the same. If an attacker has access to codecov github repo and is able to update hashes, he'd likely update all of them, so checking all of them seems redundant and clutters our CI file (since it isn't possible to declare a subroutine for such task).

Reply:
Ok, fixed.